docs/ReleaseNotes.md

```diff
@@ -1,6 +1,14 @@
 Release Notes for Version 14
 ============================
 
+Build 023
+-------
+Published as version 14.16.0
```

Comment:
For semantic versioning, a bug fix should only affect the 3rd level of version. So, this should be v14.15.1.

But, having said that, we haven't shipped v14.15.0 yet because of the unit test failure, so you can probably add this bug fix to that release instead.